influxql/ast.go

```diff
@@ -857,6 +859,48 @@ func (s *SelectStatement) RewriteDistinct() {
 	}
 }
 
+// ColumnNames will walk all fields and functions and return the appropriate field names for the select statement
+// while maintaining sort order of the field names
```

Comment:
Just so I am clear, it will maintain the order of whatever is in `s.Fields`? 'cos I see no sorting going on in this function.